src/data/ACTIONS/root/LIMIT_BREAK.ts

```diff
@@ -0,0 +1,194 @@
+import {ensureActions} from '../type'
+
+const icon = 'https://xivapi.com/i/000000/000103.png'
+
+export const LIMIT_BREAK = ensureActions({
+	// Tank LB1
+	SHIELD_WALL: {
+		id: 197,
+		name: 'Shield Wall',
+		icon,
+	},
+
+	// Tank LB2
+	STRONGHOLD: {
+		id: 198,
+		name: 'Stronghold',
+		icon,
+	},
+
+	// PLD LB3
+	LAST_BASTION: {
+		id: 199,
+		name: 'Last Bastion',
+		icon,
+	},
+
+	// WAR LB3
+	LAND_WAKER: {
+		id: 4240,
+		name: 'Land Waker',
+		icon,
+	},
+
+	// DRK LB3
+	DARK_FORCE: {
+		id: 4241,
+		name: 'Dark Force',
+		icon,
+	},
+
+	// GNB LB3,
+	GUNMETAL_SOUL: {
+		id: 17105,
+		name: 'Gunmetal Soul',
+		icon,
+	},
+
+	// Healer LB1
+	HEALING_WIND: {
+		id: 206,
+		name: 'Healing Wind',
+		icon,
+	},
+
+	// Healer LB2
+	BREATH_OF_THE_EARTH: {
+		id: 207,
+		name: 'Breath of the Earth',
+		icon,
+	},
+
+	// WHM LB3
+	PULSE_OF_LIFE: {
+		id: 208,
+		name: 'Pulse of Life',
+		icon,
+	},
+
+	// SCH LB3
+	ANGEL_FEATHERS: {
+		id: 4247,
+		name: 'Angel Feathers',
+		icon,
+	},
+
+	// AST LB3
+	ASTRAL_STASIS: {
+		id: 4248,
+		name: 'Astral Stasis',
+		icon,
+	},
+
+	// Melee LB1
+	BRAVER: {
+		id: 200,
+		name: 'Braver',
+		icon,
+	},
+
+	// Melee LB2
+	BLADEDANCE: {
+		id: 201,
+		name: 'Bladedance',
+		icon,
+	},
+
+	// MNK LB3
+	FINAL_HEAVEN: {
+		id: 202,
+		name: 'Final Heaven',
+		icon,
+	},
+
+	// DRG LB3
+	DRAGONSONG_DIVE: {
+		id: 4242,
+		name: 'Dragonsong Dive',
+		icon,
+	},
+
+	// NIN LB3
+	CHIMATSURI: {
+		id: 4243,
+		name: 'Chimatsuri',
+		icon,
+	},
+
+	// SAM LB3
+	DOOM_OF_THE_LIVING: {
+		id: 7861,
+		name: 'Doom of the Living',
+		icon,
+	},
+
+	// Ranged LB1
+	BIG_SHOT: {
+		id: 4238,
+		name: 'Big Shot',
+		icon,
+	},
+
+	// BRD LB3
+	SAGITTARIUS_ARROW: {
+		id: 4244,
+		name: 'Sagittarius Arrow',
+		icon,
+	},
+
+	// MCH LB3
+	SATELLITE_BEAM: {
+		id: 4245,
+		name: 'Satellite Beam',
+		icon,
+	},
+
+	// DNC LB3
+	CRIMSON_LOTUS: {
+		id: 17106,
+		name: 'Crimson Lotus',
+		icon,
+	},
+
+	// Ranged LB2
+	DESPERADO: {
+		id: 4239,
+		name: 'Desperado',
+		icon,
+	},
```

Comment:
Should this maybe be in the order of LB1, LB2, BRD/MCH/DNC LB3?

Reply:
Yup. Typo.